spec/helpers/acceptance_spec_helper.rb

```diff
@@ -26,14 +26,21 @@ def document_media_requests
 # @option opts [Hash]    :file_exists            (nil) Check if file exists
 # @option opts [Class]   :expected_error_class   (nil) The expected error class
 # @option opts [Regexp]  :expected_error_regexp  (nil) The expected error regular expression
+# @param [Proc] opts_mod an optional block that is called when rspec is running - allows dynamic changing of opts with
+#    access to rspec context (i.e. let and let! values)
 # @return [void]
-def standard_request_options(http_method, description, expected_status, opts = {})
+def standard_request_options(http_method, description, expected_status, opts = {}, &opts_mod)
```

Comment:
I'm not so sure about this proc - a block might be better. I'll have a look at it.